.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: go
+go:
+  - 1.5
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -qq libgl1-mesa-dev xorg-dev
+install:
+  - go get golang.org/x/tools/cmd/vet
+script:
+  - go get -t -v ./v3.1/...
+  - diff -u <(echo -n) <(gofmt -d -s .)
```

Comment:
In other words, the reason I think this doesn't deserve a comment is because it is the minimum (known to me) set of commands to check for `gofmt` issues. If this line can be made any simpler, we should do that.

Since it mentions `gofmt` and is the simplest possible command, I don't think a comment is needed.